__tests__/components/editor/InputLookup.test.js

```diff
@@ -26,7 +26,9 @@ const plProps = {
 }
 
 describe('<InputLookup />', () => {
-  const wrapper = shallow(<InputLookup {...plProps} />)
+  // our mock formData function to replace the one provided by mapDispatchToProps
+  const mockFormDataFn = jest.fn()
+  const wrapper = shallow(<InputLookup.WrappedComponent {...plProps} handleSelectedChange={mockFormDataFn} />)
```

Comment:
it would be nice to add a test, that calls the mockFormDataFn() so you know `handleSelectedChange` is actually being called.
what do you think?

Reply:
🆗